stripe/src/main/java/com/stripe/android/paymentsheet/ui/BaseSheetActivity.kt

```diff
@@ -93,7 +93,7 @@ internal abstract class BaseSheetActivity<ResultType> : AppCompatActivity() {
             }
         }
 
-        updateToolbarButton(true)
+        updateToolbarButton(supportFragmentManager.backStackEntryCount == 0)
```

Comment:
Is this change expected?

Reply:
Yes. `supportFragmentManager` might already have a backStack when the activity is being recreated.